k8s/hyperconverged/tutorial-configure-openebs-gke.md

```diff
@@ -112,10 +117,39 @@ cd openebs/k8s
 Apply OpenEBS Operator and add related OpenEBS Storage Classes, that can then be used by developers/apps:
 
 ```bash
-kubectl apply -f openebs-operator.yaml
-kubectl apply -f openebs-storageclasses.yaml
+root@ubuntu:~/demo_k8/openebs/k8s$ kubectl apply -f openebs-operator.yaml
+namespace "openebs" created
+serviceaccount "openebs-maya-operator" created
+clusterrole.rbac.authorization.k8s.io "openebs-maya-operator" created
+clusterrolebinding.rbac.authorization.k8s.io "openebs-maya-operator" created
+deployment.apps "maya-apiserver" created
+service "maya-apiserver-service" created
+deployment.apps "openebs-provisioner" created
+deployment.apps "openebs-snapshot-controller" created
+customresourcedefinition.apiextensions.k8s.io "storagepoolclaims.openebs.io" created
+customresourcedefinition.apiextensions.k8s.io "storagepools.openebs.io" created
+storageclass.storage.k8s.io "openebs-standard" created
+storageclass.storage.k8s.io "snapshot-promoter" created
+customresourcedefinition.apiextensions.k8s.io "volumepolicies.openebs.io" created
+
+root@ubuntu:~/demo_k8/openebs/k8s$ kubectl get pods -n openebs
+NAME                                           READY     STATUS    RESTARTS   AGE
+maya-apiserver-84fd4f776d-w5g9j                1/1       Running   0          47s
+openebs-provisioner-74cb999586-v7jjm           1/1       Running   0          45s
+openebs-snapshot-controller-6449b4cdbb-t7m29   2/2       Running   0          44s
+```
+```
+root@ubuntu:~/demo_k8/openebs/k8s$ kubectl apply -f openebs-storageclasses.yaml
+storageclass.storage.k8s.io "openebs-standalone" created
+storageclass.storage.k8s.io "openebs-percona" created
+storageclass.storage.k8s.io "openebs-jupyter" created
+storageclass.storage.k8s.io "openebs-mongodb" created
+storageclass.storage.k8s.io "openebs-cassandra" created
+storageclass.storage.k8s.io "openebs-redis" created
+storageclass.storage.k8s.io "openebs-kafka" created
+storageclass.storage.k8s.io "openebs-zk" created
+storageclass.storage.k8s.io "openebs-es-data-sc" created
 ```
-
 Note: The persistent storage is carved out from the space available on the nodes (default host directory : /var/openebs). There are efforts underway to provide administrator with additional options of consuming the storage (as outlined in "openebs-config.yaml"). These are slated to work hand-in-hand with the local storage manager of the kubernetes that is due in Kubernetes 1.7/1.8.
```

Comment:
**Note:** ...default host directory: /var/openebs)
"openebs-config.yaml" = *openebs-config.yaml*
with the local Kubernetes storage manager ...